examples/create_machine_pool/README.md

```diff
@@ -0,0 +1,69 @@
+# Creating a Machine Pool for existing Openshift cluster
+
+This Terraform example creates a Node Pool for existing OCM cluster. Creating new Machine pool allows users to add different machine types in the cluster..
```

Comment:
`This Terraform example creates a **Machine Pool** for existing OCM cluster. Creating new Machine pool allows users to add different machine types in the cluster.`